bridge/engine/ProcessMissingVideoPacketsJob.cpp

```diff
@@ -27,14 +27,14 @@ void ProcessMissingVideoPacketsJob::run()
 {
     auto timestamp = utils::Time::getAbsoluteTime();
     auto* videoMissingPacketsTracker = _ssrcContext.videoMissingPacketsTracker.get();
-    if (!videoMissingPacketsTracker || !videoMissingPacketsTracker->shouldProcess(timestamp / 1000000ULL))
+    if (!videoMissingPacketsTracker || !videoMissingPacketsTracker->shouldProcess(timestamp))
```

Comment:
I prefer if we are consistently using the absolute timestamp. Also these divisions will fail if the timestamp would start at a point that wraps soon.